README.md

```diff
@@ -29,15 +57,36 @@ The Pieces Copilot Streamlit Bot is an interactive chatbot application built usi
    ```
    pip install -r requirements.txt
    ```
-
-4. Run the app
+3. Run the app
 
    ```python
    streamlit run pieces-bot.py
    ```
+## Additional Resources
+
+- [Pieces OS Python SDK](https://github.com/pieces-app/pieces-os-client-sdk-for-python)
+- [Pieces Documentation](https://docs.pieces.app/build/reference/python/)
+- [Streamlit](https://streamlit.io/)
+- [Websocket](https://websockets.readthedocs.io/en/stable/)
+
+## Contributing
+
+We welcome your contributions to Pieces! All changes (even with commit access) need a pull request & thumbs-up from a core dev
```

Comment:
I would tag add mine and shivay's github links to this in replacement of `core dev`. Also, I'd recommend that we mention that a fork is required. Then they should open a pr for their fork to get reviewed.